hugo/content/en/_index.html

```diff
@@ -13,6 +13,7 @@ <h1 class="section__title">alpha.cuelang.org is currently in active development<
 
 {{< logos
     title="Users of CUE"
+    intro="Organisations in every industry use CUE to power their software and services"
```

Comment:
@JorindeUsMedia spelling nit: should match the one seen below: Organizations.

Reply:
Changed it :)